packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -68,6 +74,8 @@ export function GridRow(props: GridRowProps) {
     isFunction(getRowClassName) && getRowClassName(apiRef!.current.getRowParams(id));
   const cssClasses = clsx(className, rowClassName, classes?.row, {
     'Mui-selected': selected,
+    [`${GRID_CSS_CLASS_PREFIX}-row--editing`]: apiRef.current.getRowMode(id) === 'edit',
+    [`${GRID_CSS_CLASS_PREFIX}-row--editable`]: editMode === 'row',
```

Comment:
Should we move those in the classes constant? Or better yet with GridClasses from here #2320